editor/editor_file_system.cpp

```diff
@@ -1965,18 +1989,18 @@ void EditorFileSystem::_reimport_file(const String &p_file, const HashMap<String
 		f->store_line("uid=\"" + ResourceUID::get_singleton()->id_to_text(uid) + "\""); //store in readable format
 
 		if (err == OK) {
-			if (importer->get_save_extension().is_empty()) {
+			if (save_extension.is_empty()) {
 				//no path
 			} else if (import_variants.size()) {
 				//import with variants
 				for (const String &E : import_variants) {
-					String path = base_path.c_escape() + "." + E + "." + importer->get_save_extension();
+					String path = base_path.c_escape() + "." + E + "." + save_extension;
```

Comment:
if it helps, you can also use the vformat macro.
```suggestion
					String path = base_path.c_escape() + vformat(".%s.%s", E, save_extension);
```